network/junos/junos_config.py

```diff
@@ -32,6 +32,13 @@
     configuration and only changes are pushed to the device.
 extends_documentation_fragment: junos
 options:
+  mode:
+    description:
+      - By default you'll work in a shared candidate configuration. The 
+        C(mode) argument can be set to either private, dynamic, batch or
+        exclusive.
+    required: false
+    default: None
```

Comment:
Can you please add:

`version_added: 2.2`

This will fix the Travis error and will mean the generated documentation has this information see `filename` and `mode` parameter in http://docs.ansible.com/ansible/apt_repository_module.html